presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -165,6 +165,16 @@ public void testSelect()
         assertSelect(TABLE_ALL_TYPES_PARTITION_KEY, false);
     }
 
+    @Test
+    public void testCreateTable()
```

Comment:
Can you please name this test in a way that captures what is the functionality tested here. IMO `testInsertToTableWithHiddenId` would be better name.